### PR TITLE
Add depth to binary creation

### PR DESCRIPTION
**Pull request by pablogs9**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -29,7 +29,7 @@ endif()
 
 project(microxrcedds VERSION "2.0.0")
 
-set(_client_tag develop)
+set(_client_tag feature/add_history_depth)
```
Comment (pablogs9) [inline CMakeLists.txt:32]: When https://github.com/eProsima/Micro-XRCE-DDS-Client/pull/266 merged:
```suggestion
set(_client_tag develop)
```

**Comment by richiprosima** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/Micro-XRCE-DDS%20Manual%20Linux/256/)
* Windows [Build Status](http://jenkins.eprosima.com:8080/job/Micro-XRCE-DDS%20Manual%20Windows/275/)

**Comment by richiprosima** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/Micro-XRCE-DDS%20Manual%20Linux/257/)
* Windows [Build Status](http://jenkins.eprosima.com:8080/job/Micro-XRCE-DDS%20Manual%20Windows/276/)

**Review by Acuadros95 — APPROVED** [review]

**Review by Acuadros95 — APPROVED** [review]

**Comment by richiprosima** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/Micro-XRCE-DDS%20Manual%20Linux/260/)
* Windows [Build Status](http://jenkins.eprosima.com:8080/job/Micro-XRCE-DDS%20Manual%20Windows/279/)

**Pull request merged**